fcrepo-http-api/src/test/java/org/fcrepo/integration/http/api/PrefersLdpIT.java

```diff
@@ -0,0 +1,704 @@
+        generateResources();
+        final var getReq = new HttpGet(mainResource);
+        getReq.setHeader("Prefer", preferLink(PREFER_MINIMAL_CONTAINER.getURI(), null));
+        try (final var dataset = getDataset(getReq)) {
+            final var graph = dataset.asDatasetGraph();
+            assertTrue(graph.contains(userRdfTriple));
+            assertTrue(graph.contains(systemManagedPropertyTriple));
+            assertFalse(graph.contains(containmentTriple));
+            assertFalse(graph.contains(membershipTriple));
+            assertFalse(graph.contains(inboundReferenceTriple));
+            assertFalse(graph.contains(embededTriple));
+        }
+    }
+
+    @Test
+    public void testPreferOmitMinimal() throws Exception {
+        generateResources();
+        final var getReq = new HttpGet(mainResource);
+        getReq.setHeader("Prefer", preferLink(null, PREFER_MINIMAL_CONTAINER.getURI()));
+        try (final var dataset = getDataset(getReq)) {
+            final var graph = dataset.asDatasetGraph();
+            assertFalse(graph.contains(userRdfTriple));
+            assertFalse(graph.contains(systemManagedPropertyTriple));
+            assertTrue(graph.contains(containmentTriple));
+            assertTrue(graph.contains(membershipTriple));
+            assertFalse(graph.contains(inboundReferenceTriple));
+            assertFalse(graph.contains(embededTriple));
+        }
+    }
+
+    @Test
+    public void testPreferIncludeMinimalAndSystem() throws Exception {
+        generateResources();
+        final var getReq = new HttpGet(mainResource);
+        getReq.setHeader("Prefer", preferLink(
+                PREFER_MINIMAL_CONTAINER.getURI() + " " + PREFER_SERVER_MANAGED.getURI(),
+                null));
+        try (final var dataset = getDataset(getReq)) {
+            final var graph = dataset.asDatasetGraph();
+            assertTrue(graph.contains(userRdfTriple));
+            assertTrue(graph.contains(systemManagedPropertyTriple));
+            assertFalse(graph.contains(containmentTriple));
+            assertFalse(graph.contains(membershipTriple));
+            assertFalse(graph.contains(inboundReferenceTriple));
+            assertFalse(graph.contains(embededTriple));
+        }
+    }
+
+    @Test
+    public void testPreferOmitSystem() throws Exception {
+        generateResources();
+        final var getReq = new HttpGet(mainResource);
+        getReq.setHeader("Prefer", preferLink(
+                null,
+                PREFER_SERVER_MANAGED.getURI()));
+        try (final var dataset = getDataset(getReq)) {
+            final var graph = dataset.asDatasetGraph();
+            assertTrue(graph.contains(userRdfTriple));
+            assertFalse(graph.contains(systemManagedPropertyTriple));
+            // Containment is consider system managed.
+            assertFalse(graph.contains(containmentTriple));
+            assertTrue(graph.contains(membershipTriple));
```

Comment:
It seems that "membership" is just as "server-managed" as "containment", no?
..which argues for omitting "membership" triples here as well.

Reply:
I was going off of past tests. As there is a specific tag for including or excluding containment I think removing them them from the "system managed" category would be clearer. But 🤷